obcsim/RS232.cpp

```diff
@@ -52,6 +52,24 @@ void RS_read(msp_link_t *lnk)
         Serial.println("--------------------------------------");
       }
       break;
+    case CMD_SEND_NVM_CITIROC_CONF:
+      Serial.println("CMD_SEND_NVM_CITIROC_CONF received");
+      len = 144;
+      if (fill_commanddata(len)) {
+        Serial.println("-------- Invoking SEND_NVM_CITI_CONF -----");
+        invoke_send(lnk, MSP_OP_SEND_NVM_CITI_CONF, commanddata, len, BYTES);
+        Serial.println("------------------------------------------");
+      }
+      break;
+    case CMD_SELECT_NVM_CITIROC_CONF:
+      Serial.println("CMD_SELECT_NVM_CITIROC_CONF received");
+      len = 1;
+      if (fill_commanddata(len)) {
+        Serial.println("-------- Invoking APPLY_NVM_CITIROC_CONF -----");
```

Comment:
Should be `MSP_OP_SELECT_CUBES_CITI_CONF`, to be in concordance to the definition in cubes-fw. (Sorry, I guess I missed this last time.)